ecchronos-binary/src/bin/ecc_schedule.py

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+''''exec python -B -- "$0" ${1+"$@"} # '''
+# vi: syntax=python
+#
+# Copyright 2022 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import print_function
+
+import os
+import sys
+from argparse import ArgumentParser
+
+try:
+    from ecchronoslib import rest, table_formatter
+except ImportError:
+    SCRIPT_DIR = os.path.dirname(__file__)
+    LIB_DIR = os.path.join(SCRIPT_DIR, "..", "pylib")
+    sys.path.append(LIB_DIR)
+    from ecchronoslib import rest, table_formatter
+
+def convert_repair_job(repair_job):
+    entry = [repair_job.job_id, repair_job.keyspace, repair_job.table, repair_job.status,
+             repair_job.get_repair_percentage(), repair_job.get_last_repaired_at(), repair_job.get_next_repair(),
+             repair_job.recurring]
+
+    return entry
+
+def print_repair_job(repair_job):
+    repair_jobs_table = [["Id", "Keyspace", "Table", "Status", "Repaired(%)",
+                          "Completed at", "Next repair", "Recurring"]]
+    repair_jobs_table.append(convert_repair_job(repair_job))
+    table_formatter.format_table(repair_jobs_table)
+
+
+def parse_arguments():
+    parser = ArgumentParser(description='Schedule a repair')
+    parser.add_argument('-k', '--keyspace', type=str,
```

Comment:
There are a few other options that might be interesting here.
For example, partitioner--range as noted in #158.
Would be interesting to add but would require a far more invasive change. Certainly an improvement for the future to think about.

Reply:
Agree, next up is merging the commands. After that i will see if i can fix all the issues mentioned in #158